__tests__/pages/data-validation_tests.js

```diff
@@ -87,4 +87,14 @@ describe("DataValidation", () => {
     props.benefits[0].benefitPageFr = "";
     expect(mountedDataValidation().html()).toContain("Fail");
   });
+
+  it("fails if a benefit does not have any linked Needs", () => {
+    props.benefits[0].benefitPageFr = "";
```

Comment:
Oh,, I though there was already a test for no linked needs - how'd I not complain about this last time? :)

Thought your test here is just deleting the French link rather than the needs column.